File: Hackerrank/if-Else.java
import java.util.*;
// Here is the link to the Problem .
// https://www.hackerrank.com/challenges/java-if-else/problem

class Solution {

    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter the Number");
        int N = scanner.nextInt();


        if ((N>=1) && (N<=100))
        {
            if (N % 2 != 0)
            {
                System.out.println("Weird");
            }

            if (N % 2 == 0)
            {

                if ((N > 2) && (N < 5)) {
                    System.out.println("Not Weird");
                } else if ((N > 6) && (N <= 20)) {
                    System.out.println("Weird");

                } else if ((N > 20)) {
                    System.out.println("Not Weird");

                } else {
                    System.out.println("yes");
                }
            }


        }
    }
}

File: Hackerrank/if-else.py
https://www.hackerrank.com/challenges/py-if-else/problem?isFullScreen=true

#!/bin/python3

import math
import os
import random
import re
import sys



if __name__ == '__main__':
    n = int(input().strip())

if n%2 != 0: # divide input number by 2 / if not equal to 0, number is odd
    print("Weird")
elif n%2 ==0 and n >2 and n<5 or n>20: # if equal to 0, number is even
    print("Not Weird")
else:
    print("Weird")
